src/Http/Parser/AuthHeaders.php

```diff
@@ -50,11 +50,19 @@ public function parse(Request $request)
     {
         $header = $request->headers->get($this->header) ?: $this->fromAltHeaders($request);
 
-        if ($header) {
-            $start = strlen($this->prefix);
+        if ($header !== null) {
+            $position = strripos($header, $this->prefix);
 
-            return trim(substr($header, $start));
+            if ($position !== false) {
+                $header = substr($header, $position + strlen($this->prefix));
+
+                return trim(
+                    strpos($header, ',') !== false ? strstr($header, ',', true) : $header
+                );
```

Comment:
Is this checking if there are other values separated by ','? 
If this is the case, it would be great to add an additional test for this.

Reply:
If the token contains a comma, the string before the first comma will be returned.

Ex:
If the given token is
`Bearer first,second`
the parser will return
`first` and will ignore the rest of the string.

Should I add additional test cases?

Reply:
Yes, it would be great if you could add a test for this!